
<launch>

  <!-- parameter tuning file -->
  <arg name="params" default="$(find crane_plus_control)/config/parameter_tuning.yaml"/>

  <node pkg="crane_plus_control" name="parameter_tuning" type="parameter_tuning.py" output="screen" required="true">
    <rosparam command="load" file="$(arg params)"/>
    <param name="target" type="str" value="$(arg target)" />
  </node>
</launch>
